azerothcore/azerothcore-wotlk
# fix(DB/Quest): Fixed Quest "Shadoweaver"

## Changes Proposed:
-  Imported commit from https://github.com/TrinityCore/TrinityCore/issues/22557

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7461
- Closes https://github.com/chromiecraft/chromiecraft/issues/1469

## SOURCE:

https://tbc.wowhead.com/quest=3379/shadoweaver
https://wowpedia.fandom.com/wiki/Shadoweaver?oldid=2072967

## Tests Performed:

- Have tailoring at 230 and at least lvl 40.
- .go c 5579
- Take quest and see the gossip for raze to follow
- .go c 5768 and check that they move now patrolling the zone

Quick test

Roaming enemies

https://streamable.com/ce95lk

Quest taking

https://streamable.com/ruyp0l

Killing them

https://streamable.com/z9isss

Maybe there is something weird on the Staff of command action, because i can see the shields on them, but he wont change aggro to the one ive selected. Could be blizz or cluld be something wrong with the item.

## How to Test the Changes:


1. Have tailoring at 230 and at least lvl 40.
2. .go c 5579
3. Take quest and see the gossip for raze to follow
4.  .go c 5768 and check that they move now patrolling the zone


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
